sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/api/UserSteps.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2022 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+package org.sormas.e2etests.steps.api;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import cucumber.api.java8.En;
+import java.io.FileReader;
+import javax.inject.Inject;
+import org.json.simple.parser.JSONParser;
+import org.sormas.e2etests.entities.services.api.UserApiService;
+import org.sormas.e2etests.helpers.api.sormasrest.UserHelper;
+import org.sormas.e2etests.helpers.environmentdata.manager.EnvironmentManager;
+import org.sormas.e2etests.state.ApiState;
+import org.testng.asserts.SoftAssert;
+
+public class UserSteps implements En {
+
+  private String responseBody;
+  private String regexUpdatedResponseBody;
+  private String expectedUserRights;
+
+  @Inject
+  public UserSteps(
+      UserHelper userHelper, UserApiService userApiService, SoftAssert softly, ApiState apiState) {
+
+    When(
+        "API:I Login into Environment",
+        () -> {
+          EnvironmentManager loginIntoEnvironment = userApiService.loginIntoEnvironment();
+        });
+
+    When(
+        "API: I get response with user permission rights",
```

Comment:
You do not use this method later in your code, so I recommend to delete it 😺

Reply:
Done. It was test method that I forget to remove